### DISCUSSION TITLE
Is it possible to statically link multiple versions of curl in same project (with prefixing?)

### DISCUSSION THREAD

**Discussion by matt1309** [curl/curl#13305] [Q&A] [answered]
I would like to be able to use libcurl with boringssl and libcurl with nss in same program.

Is there any of I could use prefixing or some other technique to avoid linker conflicts?

**Comment by bagder** [comment, marked as answer]
You can already build libcurl with multiple TLS libraries. Just specify more than one with configure. You use `curl_global_sslset` to decide which to use at run-time.
